File: src/main/java/org/usfirst/frc/team1458/lib/core/BlankAutoMode.java
package org.usfirst.frc.team1458.lib.core;

/**
 * Blank autonomous mode
 *
 * @author asinghani
 */
public class BlankAutoMode implements AutoMode {
	@Override
	public String getName() {
		return "Blank Autonomous Mode";
	}

	@Override
	public String toString() {
		return getName();
	}

	@Override
	public void auto() {
		// Do nothing
	}
}

File: 1111 JAVA STUFF THAT DOESNT WORK/main/java/org/usfirst/frc/team1458/lib/core/BlankAutoMode.kt
package org.usfirst.frc.team1458.lib.core

/**
 * Blank autonomous mode
 *
 * @author asinghani
 */
class BlankAutoMode : AutoMode {
    override val name: String
        get() = "Blank Autonomous Mode"

    override fun toString(): String {
        return name
    }

    override fun auto() {
        // Do nothing
    }
}